metadata/junit/junit.go

```diff
@@ -102,6 +102,7 @@ type Result struct {
 	Failure    *string     `xml:"failure,omitempty"`
 	Output     *string     `xml:"system-out,omitempty"`
 	Error      *string     `xml:"system-err,omitempty"`
+	Errored    *string     `xml:"error,omitempty"`
```

Comment:
I think you want to process this in the Result.Truncate() function too.

(Also wondering how easy it is to rename 'Output' and 'Error' to append 'Sys' or 'System', but think that can be a separate PR.

Reply:
I will add the truncate, about the renaming, it will break compatibility, but If that's not a problem I can try to rename it.

Reply:
I think leaving it to a separate PR (or filing an issue about it) is fine.

Also forgot, but you'll need to update Message() as well.

Reply:
Done